http-cookie.gemspec

```diff
@@ -18,7 +18,11 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/sparklemotion/http-cookie"
   gem.license       = "MIT"
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = Dir.glob('lib/**/*') + [
```

Comment:
I would prefer to be explicit about the files we're including. `Dir.glob` risks pulling in files accidentally when the gem is packaged. Please either list the individual files or constrain `git ls-files` to the `lib` directory.